model_compression_toolkit/core/pytorch/pytorch_implementation.py

```diff
@@ -118,16 +118,20 @@ def to_tensor(self, tensor: Any) -> torch.Tensor:
 
     def model_reader(self,
                      module: Module,
-                     representative_data_gen: Callable) -> Graph:
+                     representative_data_gen: Callable,
+                     model_leaf_layers: list = None) -> Graph:
         """
         Convert a framework's module into a graph.
         Args:
             module: Framework's module.
             representative_data_gen (Callable): Dataset used for calibration.
+            model_leaf_layers (list): List of the module's custom layers, these layers shouldn't be divided into
```

Comment:
List of the module's custom layers. These layers shouldn't be divided into their submodules. Please note that their quantization will not be optimized.